server/app.ts

```diff
@@ -54,6 +55,7 @@ export default function createApp(services: Services): express.Application {
   app.use('/change-establishment', establishmentRoutes(services))
   app.use('/prisoner', prisonerRoutes(services))
   app.use('/search', searchRoutes(services))
+  app.use('/altSearch', altSearchRoutes(services))
```

Comment:
Should this be `alt-search` to follow the standard?

Reply:
apologies Paul, had a conversation with Tom about this - Did for simplicity sake (to make find and replace only touch files imports/addresses in one go for testing purposes) this is a temporary url/file naming and is addressed in my next PR

^ to all comments